testing/src/main/java/io/airlift/testing/Closeables.java

```diff
@@ -11,98 +13,92 @@ private Closeables()
     {
     }
 
-    public static void closeQuietly(Closeable... closeables)
+    public static void closeQuietly(AutoCloseable... closeables)
```

Comment:
let's try to avoid this method -- https://prestosql.slack.com/archives/CP1MUNEUX/p1586372796137000